platform/accessibility/README.md

```diff
@@ -1,24 +1,77 @@
+* **Define:** We help in early project discussions to make sure inclusive design is woven into your work. Main areas to consider at this phase include:
+   * challenges with user flows
+   * color contrast
+   * color blindness
+   * accessible interactions
+   * design for various device sizes
+   * cognitive impact
+* **Discover:** We provide more specific feedback at this phase, including:
+   * relevant design system components and patterns
+   * feedback on recruiting diverse research participants
+   * additional testing scenarios
+* **Build:** We review your completed test plan and automated end-to-end test results, manually test the happy path, and triage issues for severity. We will let you know whether issues must be fixed pre- or post-launch. This testing is done as part of the VSP staging review.
+* **Validate:** Post launch, we do a full accessibility audit and ask the VA 508 office to conduct a review. Post-launch audits uncover usability issues that might not otherwise appear on an automated report. These findings drive future innovation, updates to the design system, and improve research sessions. 
+(Note: This audit was previously done prior to launch, but is now done post-launch to emphasize speed and agility.)
+
+## Resources and guides
+**General Guidance**
+* [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md) This checklist provides a good overview of accessibility requirements and tools that will help you in early-stage testing.
+* [Accessibility Best Practices:](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md) Learn about what areas of your product will be evaluated for accessibility and inclusivity.
 * [Accessibility Dev Review Step-by-Step](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/accessibility-dev-review-step-by-step.md)
-* [Disability Considerations](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/disability-considerations.md)
-* [Examples of Assistive Tech](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/examples-of-assistive-tech.md_)
+* [Examples of Assistive Tech](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/examples-of-assistive-tech.md)
 * [Inclusive Design Guidelines](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/inclusive-design-guidelines.md)
+* [User Stories](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/user-stories.md)
```

Comment:
Perhaps Inclusive User Stories?  I know I had it as user stories, but reading it now it isn't as clear these are specific to a11y / inclusive design.